.github/workflows/main.yml

```diff
@@ -6,11 +6,9 @@ on:
   push:
     branches:
       - master
-    tags:
-      - v*
   pull_request:
     branches:
-      - master
+      - '*'

```

Comment:
Why do we want to change this?
It seems to me we only want to run our jobs on MR which are targeting our main branch.

Reply:
I change this to start workflow foreach pull request whatever branch.
Example: workflow doesn't triggers for pr #179